components/style/gecko/rules.rs

```diff
@@ -67,6 +68,16 @@ impl ToNsCssValue for font_feature_settings::T {
     }
 }
 
+impl ToNsCssValue for font_language_override::SpecifiedValue {
+    fn convert(self, nscssvalue: &mut nsCSSValue) {
+        match self {
+            font_language_override::SpecifiedValue::Normal => nscssvalue.set_normal(),
+            font_language_override::SpecifiedValue::Override(ref lang) => nscssvalue.set_string(&*lang),
+            font_language_override::SpecifiedValue::System(_) => {},
```

Comment:
Ok, so the thing is "this can't appear here because the descriptor is only specified by the user", right?

If so, maybe leave a comment, and mark it as `unreachable!`?